chia/daemon/client.py

```diff
@@ -14,11 +14,10 @@
 
 
 class DaemonProxy:
-    def __init__(self, uri, ssl_context):
+    def __init__(self, uri: str, ssl_context: Optional[ssl.SSLContext]):
```

Comment:
Is optional necessary?

Reply:
Since the caller: https://github.com/aisk/chia-blockchain/blob/more-types/chia/daemon/client.py#L103-L108 have the optianl, so it's necessary here.